packages/atomic/src/utils/utils.ts

```diff
@@ -39,6 +39,13 @@ export function sanitize(string: string) {
   return string.replace(reg, (match) => map[match]);
 }
 
+export function filterProtocol(uri: string) {
```

Comment:
Should we do this at the controller level instead? I'm thinking that this issue could affect a lot of clients, and this could be useful.

However, a counterpoint is that if we do it ourselves at the controller level, we could forget some fields (e.g., `Product.thumbnail`) that would also need to be protected, and we need clients to stay vigilant against XSS too.

Either way, I believe adding a check against XSS to the React samples would be good practice, so I'll create an issue for that.